downstream/modules/builder/con-container_file.adoc

```diff
@@ -2,13 +2,10 @@
 
 = Containerfile
 
-Once your definition file is created, Ansible Builder reads and validates it, then creates a `Containerfile`, and finally passes the `Containerfile` to Podman to package and create your automation execution environment image using the following instructions:
+Once your definition file is created, {Builder} reads and validates it, creates a `Containerfile` and container build context, and optionally passes these to Podman to build your {ExecEnvNameSing} image. The container build occurs in several distinct stages: `base` , `galaxy`, `builder`, and `final`. The image build steps (along with any corresponding custom `prepend_` and `append_` steps defined in `additional_build_steps`) are:
```

Comment:
Suggest changing "Once" to "After". According to Style Guide: Use once only to mean "one time". Do not use as a conjunction to mean "after" or "when".